packages/@okta/vuepress-site/docs/guides/add-logingov-idp/main/index.md

```diff
@@ -5,33 +5,33 @@ meta:
     content: Okta supports authentication with Login.gov as an external Identity Provider. Set up the Login.gov IdP using OpenID Connect with private key JWT in Okta.
 ---
 
-This document explains how to configure Login.gov as an [external Identity Provider](/docs/concepts/identity-providers/) for your application by creating an Identity Provider in Okta, creating an application at Login.gov, testing the configuration, and creating a Login.gov sign-in option.
+This document explains how to configure Login.gov as an [external Identity Provider](/docs/concepts/identity-providers/) for your app. You do this by creating an Identity Provider in Okta, creating an app at Login.gov, testing the configuration, and creating a Login.gov sign-in option.
 
 ---
 
 #### Learning outcomes
 
-Configure Login.gov as an external Identity Provider so that your users can quickly sign up or sign in to your application using their Login.gov account.
+Configure Login.gov as an external Identity Provider so that your users can quickly sign up or sign in to your app using their Login.gov account.
```

Comment:
```suggestion
Configure Login.gov as an external IdP so that your users can quickly sign up or sign in to your app using their Login.gov account.
```